src/components/auth/SocialMediaLogin.test.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import 'react-testing-library/cleanup-after-each';
+import 'jest-dom/extend-expect';
+import { render, cleanup } from 'react-testing-library';
+import SocialAuthButton from './SocialAuthButton';
+import { SocialMediaLogin } from './SocialMediaLogin';
+import reducers from '../../reducers/auth';
+
+afterEach(cleanup);
+
+const props = {
+  location: {
+    search: '?token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYX…kyMX0.pVDyzSWUrr5_x3m66Wrg3rAJjbVkAp-quRg55Etlkv0',
```

Comment:
Is this a real token. If yes, please add it to the env variables

Reply:
Resolved.